package.json

```diff
@@ -29,7 +29,7 @@
     "@edx/frontend-component-footer": "^10.0.11",
     "@edx/frontend-component-header": "^2.0.5",
     "@edx/frontend-platform": "^1.5.3",
-    "@edx/paragon": "^7.2.0",
+    "@edx/paragon": "^12.0.4",
```

Comment:
I am a bit worried about this version jump. It this upgrade introduce any non-related UI changes on the program-console?

Reply:
I will double check.

Reply:
As far as I can tell, the components we're using (which were just Collapsible and StatusAlert before) aren't affected by any breaking changes.